crates/re_ui/src/lib.rs

```diff
@@ -934,7 +939,13 @@ impl ReUi {
                 .layout()
                 .align_size_within_rect(text.size(), text_rect)
                 .min;
-            text.paint_with_visuals(ui.painter(), text_pos, &visuals);
+
+            let mut text_color = visuals.text_color();
+            if unnamed_style {
+                // TODO(ab): use design tokens
+                text_color = text_color.gamma_multiply(0.5);
```

Comment:
I'm not sure we should both be doing italics _and_ transparency. Transparency also has an opportunity to be confused with a "grayed out" disabled look. What says @martenbjork ?

Reply:
Punting on this. With #4802, I will further isolate the location where this formatting happens, so should be easy to revisit.